lib/matplotlib/gridspec.py

```diff
@@ -145,7 +157,7 @@ def __getitem__(self, key):
             else:
                 if k1 < 0:
                     k1 += nrows
-                if k1 >= nrows or k1 < 0 :
+                if k1 >= nrows or k1 < 0:
```

Comment:
slightly prefers `if not 0 <= k1 < nrows:` (same below

Reply:
Didn't even know you could do that! (though I didn't write the original code)